File: judge-runtime/spec/lang.java/Validator.java
import java.util.*;

class Validator {
    public String swapFirstAndLastWords(String s) {
        String[] words = s.split(" ");

        if (words.length <= 1) {
            return s;
        }

        String temp = words[0];
        words[0] = words[words.length - 1];
        words[words.length - 1] = temp;

        return String.join(" ", words);
    }
}

File: judge-runtime/spec/lang.kotlin/validator.kt
class Validator {
    fun swapFirstAndLastWords(s: String): String {
        val words = s.split(" ").toMutableList()

        if (words.size <= 1) {
            return s
        }

        val temp = words[0]
        words[0] = words[words.size - 1]
        words[words.size - 1] = temp

        return words.joinToString(" ")
    }
}
